ncids-webpack/webpack.config.js

```diff
@@ -0,0 +1,164 @@
+
+		/**
+		 * The output configuration defines how Webpack should handle the generated
+		 * output files, such as bundled JavaScript files and other assets, and where
+		 * they should be emitted.
+		 */
+		output: {
+			// Absolute path of output directory of generated files.
+			path: path.resolve(__dirname, 'dist'),
+			// Filename pattern for generated javascript bundles.
+			filename: '[name].bundle.js',
+			// Filename pattern for generated assets.
+			assetModuleFilename: 'assets/[name][ext]',
+			// Clears out the output directory before generated new files.
+			clean: true,
+		},
+		module: {
+			rules: [
+				{
+					test: /\.(scss|css)$/,
+					use: [
+						/**
+						 * The mini-css-extract-plugin is used for production builds. It
+						 * extracts CSS into separate files. The extracted CSS files are then
+						 * used by the html-webpack-plugin to inject a <link> tag into the
+						 * HTML file during the build process.
+						 */
+						MiniCssExtractPlugin.loader,
+
+						/**
+						 * css-loader interprets and resolves css imports and dependencies
+						 */
+						{
+							loader: 'css-loader',
+							options: {
+								/*
+								 * Source maps are generated to provide mapping between compiled
+								 * code and original code for easier debugging
+								 *
+								 * This should only be turned on for dev builds.
+								 */
+								sourceMap: !isProduction,
+							},
+						},
+
+						/**
+						 * Add post css plugins, see postcss.config.js for more details.
+						 */
+						{
+							loader: 'postcss-loader',
+							options: {
+								sourceMap: !isProduction,
+							},
+						},
+
+						/**
+						 * NCIDS CSS requires compiling your Sass with load paths using
+						 * dart-sass.
+						 *
+						 * See https://sass-lang.com/documentation/at-rules/use#load-paths
+						 */
+						{
+							loader: 'sass-loader',
+							options: {
+								sassOptions: {
+									includePaths: [
+										// Includes path to /packages directory for NCIDS packages.
+										path.join(__dirname, './node_modules/@nciocpl/ncids-css/packages'),
+										// Includes path to /uswds-packages directory for USWDS packages.
+										path.join(__dirname, './node_modules/@nciocpl/ncids-css/uswds-packages'),
+									],
+								},
+								sourceMap: !isProduction,
+							},
+						},
+					],
+				},
+
+				/**
+				 * The asset/inline type rule for SVG files allows them to be handled as
+				 * inline data URLs.
+				 *
+				 * Imported SVGs will be transformed into data URLs and inlined directly
+				 * into the bundle, reducing the number of separate network requests
+				 * needed to load the SVG images.
+				 */
+				{
+					test: /\.svg/,
+					type: 'asset/inline',
+				},
+			],
+		},
+		plugins: [
+			/**
+			 * HtmlWebpackPlugin generates HTML files with injected script and link
+			 * tags, using the provided template and applies minification options if
+			 * enabled.
+			 */
+			new HtmlWebpackPlugin({
```

Comment:
Basically unless this is a Single-page Javascript application, this will not be needed in their configuration. You know how crappy it is to publish multiple web pages out using this. :)